packages/utils/src/TextUtils.ts

```diff
@@ -7,7 +7,7 @@ class TextUtils {
    * One, Two, and Three
    * @param {String[]} items The items to join in a list
    */
-  static join(items) {
+  static join(items: string[]): string {
```

Comment:
Looks like this can be `string[] | null`. Declaring it that way instead of making it optional will make the compiler warn you if you just omit the param vs. passing a possibly null param. There should be no reason for us to explicitly call the function w/o params